ownership-chain/runtime/laos/src/configs/fee/mod.rs

```diff
@@ -0,0 +1,47 @@
+use crate::PhantomData;
+use frame_support::traits::{tokens::currency::Currency, Imbalance, OnUnbalanced};
+
+pub struct DealWithFees<R>(PhantomData<R>);
+
+impl<R> DealWithFees<R>
+where
+	R: pallet_balances::Config + pallet_authorship::Config,
+{
+	// Distributes the specified imbalance to the author of the block
+	fn to_author(imbalance: pallet_balances::NegativeImbalance<R>) {
```

Comment:
don't think this deserves a separate function taking into account the comment below